docs/data-types/cassandra.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+title: Cassandra
+parent: Data Type Conversion
+nav_order: 2
+---
+
+# Schema migration for Cassandra
+{: .no_toc }
+
+Spanner migration tool makes some assumptions while performing data type conversion from Cassandra to Spanner.
+There are also nuances to handling certain specific data types. These are captured below.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Data Type Mapping
+
+The Spanner migration tool maps Cassandra primitive types to Spanner types as follows:
```

Comment:
Please add details on Adapter options, also a short note on Adapter in the intro might help a lot.
Also add a note that (something on these lines - my comment is not formal wording) we support migration to only Google SQL Dialect from Cassandra and we migrate with options which are compatible with the Cassandra Adapter.